README.md

```diff
@@ -190,6 +190,19 @@ Name | Type | Required | Description | Example
 --- | --- | --- | --- | ---
 remotePath | string | yes | The path to a folder. The path may contain environment variables that could be expanded on the server side. Due to security reasons only variables listed below would be expanded: `APPDATA`, `LOCALAPPDATA`, `PROGRAMFILES`, `PROGRAMFILES(X86)`, `PROGRAMDATA`, `ALLUSERSPROFILE`, `TEMP`, `TMP`, `HOMEPATH`, `USERPROFILE`, `PUBLIC` | `%HOMEPATH%\\SomeFolder\\` or `C:\\Users\\user\\SomeFolder\\`
 
+## Custom Path
+
+If you want to set your custom path like WAD then you can set path as follow as:  
+```
+process.env.WAD_PATH = 'C:\\Windows\\Program Files\\Windows Application Driver\\WinAppDriver.exe'
+```
+
+#### Custom paths
+Env Name | Description
+--- | ---
+WAD_PATH | WinAppDriver.exe path (Must full path)
```

Comment:
We usually prefix Appium-specific env variable with APPIUM_

Reply:
Edited.